modules/apps/commerce/commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/portlet/shared/search/CPOptionFacetsPortletSharedSearchContributor.java

```diff
@@ -97,14 +98,17 @@ public void contribute(
 				frequencyThreshold = GetterUtil.getInteger(
 					portletPreferences.getValue("frequencyThreshold", null),
 					frequencyThreshold);
+				maxOptions = GetterUtil.getInteger(
+					portletPreferences.getValue("maxOPtions", null),
```

Comment:
there's a typo here

Reply:
```suggestion
					portletPreferences.getValue("maxOptions", null),
```

Reply:
is it working also with the typo?

Reply:
I only tested with specifications, but in theory, no it will not work for options. I am fixing it now. @andrea-ale-sbarra